src/PocketMine.php

```diff
@@ -138,9 +138,9 @@ function check_platform_dependencies(){
 		$wantedVersionLock = "0.3";
 		$wantedVersionMin = "$wantedVersionLock.0";
 		if($chunkutils2_version !== false && (
-			version_compare($chunkutils2_version, $wantedVersionMin) < 0 ||
-			preg_match("/^" . preg_quote($wantedVersionLock, "/") . "\.\d+(?:-dev)?$/", $chunkutils2_version) === 0 //lock in at ^0.2, optionally at a patch release
-		)){
+				version_compare($chunkutils2_version, $wantedVersionMin) < 0 ||
```

Comment:
Revert all the changes from this file. It's unrelated from the PR.

Reply:
https://github.com/pmmp/PocketMine-MP/blob/stable/src/PocketMine.php#L141

Reply:
this is basic in the Pocketmine.php

Reply:
What do you mean?

Reply:
this is in the stable branch there is no difference

Reply:
There are differences because this is in the "Files changed" section.

Reply:
These formatting changes must be removed

Reply:
> These formatting changes must be removed

this was done in the commits